components/custom-radio-button/css/stories/states.stories.mdx

```diff
@@ -0,0 +1,46 @@
+<!-- @license CC0-1.0 -->
+
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { CustomRadioButton, defaultArgs } from "../template";
+import "../index.scss";
+
+<Meta
+  id="css-custom-radio-button"
+  title="CSS Component/Custom Radio Button/States"
+  argTypes={{}}
+  parameters={{
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+# CustomRadio Button Component states
+
+## Disabled
+
+<Canvas>
+  <Story
+    name="disabled"
+    args={{
+      ...defaultArgs,
+      disabled: true,
+    }}
+  >
+    {CustomRadioButton.bind({})}
+  </Story>
+</Canvas>
+
+## Checked
+
+<Canvas>
+  <Story
+    name="checked"
+    args={{
+      ...defaultArgs,
+      checked: true,
+    }}
+  >
+    {CustomRadioButton.bind({})}
+  </Story>
+</Canvas>
```

Comment:
I think it would be great if we'd also include the following states. Amongst other things, having those will help us with visual regression tests.

- checked _and_ disabled
- active
- focus
- invalid

Please also include those states in the ArgsTable of the default story.